tests/spcs/test_image_repository.py

```diff
@@ -131,3 +117,20 @@ def test_list_tags(
     assert json.loads(result.output) == [
         {"tag": "/DB/SCHEMA/IMAGES/super-cool-repo:1.2.0"}
     ]
+
+
+@mock.patch(
+    "snowflake.cli.plugins.spcs.image_repository.commands.ImageRepositoryManager._execute_query"
+)
+@mock.patch(
+    "snowflake.cli.plugins.spcs.image_repository.commands.ImageRepositoryManager._conn"
+)
+def test_get_repository_url_cli(mock_conn, mock_execute_query, runner, mock_cursor):
+    mock_execute_query.return_value = mock_cursor(
+        rows=MOCK_ROWS_DICT,
+        columns=MOCK_COLUMNS,
+    )
+
+    result = runner.invoke(["spcs", "image-repository", "url", "IMAGES"])
+    assert result.exit_code == 0, result.output
+    assert result.output.strip() == MOCK_ROWS_DICT[0]["repository_url"]
```

Comment:
add a test if provided with a repository name that doesn't exist (can borrow no repository found error from https://github.com/Snowflake-Labs/snowcli/pull/715)